core/src/migrations/000078-propertyKeepValueIfNotFound.ts

```diff
@@ -0,0 +1,15 @@
+export default {
+  up: async function (migration, DataTypes) {
+    await migration.sequelize.transaction(async () => {
+      await migration.addColumn("properties", "keepValueIfNotFound", {
+        type: DataTypes.BOOLEAN,
```

Comment:
This migration allows `properties.keepValueIfNotFound` to be null - is that intended?  I think it might be preferable to not allow it to be null, and in the migration, update all exiting rows to `true` to match the existing behavior

Reply:
I updated this migration, but I'm not sure about matching existing behavior. I think setting it to `true` for existing properties is definitely the safest option, that way if they don't want anything to be deleted, nothing will happen. It will however be a bit of extra work if you _do_ want to delete things, which I think may be what most people want to do.

Reply:
We can add that to the migration guide! 

Check the note on the other migration too...